source/includes/read/project.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use MongoDB\Client;
+
+$client = new Client('<connection string>');
```

Comment:
Same as https://github.com/mongodb/docs-php-library/pull/102#discussion_r1719513172
```suggestion
$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
$client = new MongoDB\Client($uri);
```